nupic/algorithms/CLAClassifier.py

```diff
@@ -558,13 +560,13 @@ def read(cls, proto):
     classifier._patternNZHistory = deque(maxlen=max(classifier.steps) + 1)
     patternNZHistoryProto = proto.patternNZHistory
     learnIteration = classifier._learnIteration - len(patternNZHistoryProto) + 1
-    for i in xrange(len(patternNZHistoryProto)):
+    for i in range(len(patternNZHistoryProto)):
```

Comment:
There are also alternative approaches to the original code to consider, too.  For example, I think the original `for i in xrange(len())` was the wrong approach.  It would have been better to use `enumerate()`, since `patternNZHistoryProto[i]` is used inside the loop.